### PR TITLE
Added R2 to destination section

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/logs/get-started/api-configuration.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ You can specify your cloud service provider destination via the required **desti
 As of May 2022, defining a unique destination for a Logpush job will no longer be required. As this constraint has been removed, you can now have more than one job writing to the same destination.
 {{</Aside>}}
 
+*  **Cloudflare R2**: endpoint name + bucket path + account ID + access key id + secret key; for example: `r2://<BUCKET_PATH>/account-id=<ACCOUNT_ID>&access-key-id=<R2_ACCESS_KEY_ID>&secret-access-key=<R2_SECRET_ACCESS_KEY>`
```
Comment (soheiokamoto) [inline content/logs/get-started/api-configuration.md]: What's endpoint name here?

Comment (angelampcosta) [reply content/logs/get-started/api-configuration.md]: I guess `r2`? I copied the example that we have here https://developers.cloudflare.com/logs/get-started/enable-destinations/r2/#manage-via-api

Comment (soheiokamoto) [reply content/logs/get-started/api-configuration.md]: I think that might have been copied from s3-compatible, but with `r2://`, I don't think there is `endpoint` parameter.

Comment (angelampcosta) [reply content/logs/get-started/api-configuration.md]: I see. So the `destination_conf` for R2 it's only `bucket path`, `account id`, `R2 access key id` and `R2 secret access key`, like this `r2://<BUCKET_PATH>/account-id=<ACCOUNT_ID>&access-key-id=<R2_ACCESS_KEY_ID>&secret-access-key=<R2_SECRET_ACCESS_KEY>`?

Comment (soheiokamoto) [reply content/logs/get-started/api-configuration.md]: Yes.

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**